test/e2e/framework/skipper/skipper.go

```diff
@@ -67,6 +66,13 @@ func InitFeatureGates(defaults featuregate.FeatureGate, overrides map[string]boo
 	return nil
 }
 
+func IsFeatureGateEnabled(feature featuregate.Feature) bool {
```

Comment:
Can you added documentation? Feature gate testing is one of those areas that have been misunderstood by test authors in the past.

I'll re-add LGTM and approve when done.

/lgtm cancel

Reply:
Thanks @pohly!
An important comment indeed. I'm also a victim of misunderstanding how it works :)

I've added the following documentation, let me know what you think:
> // IsFeatureGateEnabled can be used during e2e tests to figure out if a certain feature gate is enabled.
> // This function is dependent on InitFeatureGates under the hood. Therefore, the test must be called with a
> // --feature-gates parameter.